CHANGELOG.md

```diff
@@ -105,7 +105,7 @@ Fixes #63
 
 ### Other Changes
 
-- changelog_to_tag action - support other than "master" for the main branch name (#66)
+- changelog_to_tag action - github action ansible test improvements
```

Comment:
This is rewriting history, not a fan. It also explicitly says "branch name", there is absolutely no indication of this applying to humans.

Note, I'm not opposed to renaming the "master" branch to "main", but as long as it's "master", then that's just how it's called.

Reply:
> This is rewriting history, not a fan.

Understood, and if this were some important information for end users of the cockpit role, we would probably have done something different.  But this particular bit of information is pretty useless for end users.

> It also explicitly says "branch name", there is absolutely no indication of this applying to humans.

Not sure what you mean.

> Note, I'm not opposed to renaming the "master" branch to "main", but as long as it's "master", then that's just how it's called.

If we don't do this, we have a couple of pretty bad alternatives (for more information, see the discussion starting here https://github.com/linux-system-roles/crypto_policies/pull/54#issuecomment-1351888226)

* put a comment like `// wokeignore:rule=master` at the end of the line, so that it looks like this:
```
- changelog_to_tag action - support other than "master" for the main branch name (#66) // wokeignore:rule=master
```
This is IMO worse for humans to read than changing the line

* exclude `CHANGELOG.md` from inclusive language checking

Since this may be the second thing a user of the role reads, it IMO definitely needs to be scanned for inclusive language

Reply:
> Understood, and if this were some important information for end users of the cockpit role, we would probably have done something different. But this particular bit of information is pretty useless for end users.

Sure, just saying that I find the cure here much worse than the (perceived) disease. This might be language barrier, a "master" is something good in German (mastering a craft or an art).

> > It also explicitly says "branch name", there is absolutely no indication of this applying to humans.
> 
> Not sure what you mean.

I mean the changelog entry says "'master' for the main branch name", so it's entirely clear this does not talk about persons, but some git label. Trying to construct an ad hominem attack out of that is so far-fetched that it could safely be ignored.

>     * put a comment like `// wokeignore:rule=master` at the end of the line, so that it looks like this:

Right, this is nonsense, let's not do that.

>     * exclude `CHANGELOG.md` from inclusive language checking
> Since this may be the second thing a user of the role reads, it IMO definitely needs to be scanned for inclusive language

Right.

Anyway, please go ahead if you find this important. I don't care about it enough to start a veto/big fight over it (pick your battles), I just won't explicitly approve it.

Reply:
> please go ahead if you find this important

But then please do the actual work and rename the "master" branch to "main", otherwise this does not make any sense.

Reply:
> > please go ahead if you find this important
> 
> But then please do the actual work and rename the "master" branch to "main", otherwise this does not make any sense.

Thank you for your discussion, @martinpitt.

Unfortunately, I have no privilege to change the default branch.

May I ask you to follow [these steps](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch)? First, you are supposed to create a `main` branch from the `master` branch. Then, switch to the `main`.

Reply:
Done
```
The default branch has been renamed!

master is now named main

If you have a local clone, you can update it by running the following commands.

git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

Reply:
Thanks!

Reply:
Does anything outside of lsr-cockpit refer to "master", like other modules, build scripts, documentation, etc.?

Reply:
> Does anything outside of lsr-cockpit refer to "master", like other modules, build scripts, documentation, etc.?

It shouldn't.  Almost all of our tooling is designed to be "main branch" agnostic e.g. https://github.com/linux-system-roles/cockpit/blob/main/.github/workflows/changelog_to_tag.yml#L45, or will somehow discover or determine what is the main branch e.g. https://github.com/linux-system-roles/mssql/blob/master/.github/workflows/weekly_ci.yml#L24
and I'm hoping we can drop some/most/all? of this code once we convert everything to use `main` as the main branch name.